feature/gribi/ate_tests/get_rpc_test/README.md

```diff
@@ -20,19 +20,31 @@ Validate gRIBI Get RPC.
     198.51.100.64/26, 198.51.100.128/26 are returned. Measure latency of Get
     RPC.
 
+    *   TODO: ensure all IPEntry, NHG and NH are returned with
+        [`fib_status=PROGRAMMED`](https://github.com/openconfig/gribi/blob/08d53dffce45e942c6e7f07521c58b557984e4b7/v1/proto/service/gribi.proto#L485).
```

Comment:
```suggestion
    *   TODO: ensure all AFTEntry in the GetResponse for the IPv4Entry, NHG and NH are
        returned with [fib_status] `PROGRAMMED`.
```

And we can move the fib_status link below after the itemized list, say line 47.

```
[fib_status]: https://github.com/openconfig/gribi/blob/08d53dffce45e942c6e7f07521c58b557984e4b7/v1/proto/service/gribi.proto#L485
```

This is to avoid long inline links.